graphdatascience/model/model_alpha_proc_runner.py

```diff
@@ -9,6 +9,16 @@
 
 class ModelAlphaProcRunner(ModelResolver):
     def store(self, model: Model, failIfUnsupportedType: bool = True) -> "Series[Any]":
+        """
+        Persist a model from the Model Catalog to disk.
+
```

Comment:
We could not find this one in the generated docs.

Also the args are written in a different style (duplicating the type information)